eng/automation/changelog/src/main/java/com/azure/resourcemanager/tools/changelog/Main.java

```diff
@@ -24,6 +25,12 @@ public class Main {
     private final static String NEW_FEATURE_TITLE = "### Features Added\n\n";
 
     public static void main(String[] args) throws Exception {
+        JSONObject json = getChangelog();
+
+        System.out.println(json.toString());
```

Comment:
Is this for debug, or for function?

Reply:
`System.out.println` is for printing out result to console for our changelog python script.

`getChangelog` is just extraction of method, for test purpose.